generators/app/tasks/installDependencies.js

```diff
@@ -26,7 +26,6 @@ const DEPENDENCIES = [
 const DEV_DEPENDENCIES = [
   'babel-eslint',
   'babel-plugin-import-glob',
-  'babel-preset-react-native',
```

Comment:
Hi all,
It would be worth to have these with stable versions on the bootstrap. Otherwise, whenever a version changes, we should make a PR updating changes.
Thanks to @ignaciosantise for noticing this

Reply:
Yes @mvbattan, I agree. Already several times happened to have to do pull request to fix something.